apps/studio/next.config.js

```diff
@@ -144,6 +144,16 @@ const nextConfig = {
         destination: '/project/:ref/storage/buckets',
         permanent: true,
       },
+      {
+        source: '/project/:ref/settings/storage',
+        destination: '/project/:ref/storage/settings',
+        permanent: true,
+      },
+      {
+        source: '/project/:ref/settings/database',
+        destination: '/project/:ref/database/settings',
+        permanent: true,
+      },
```

Comment:
Redirects from the old URLs to the new. Important for any references I missed in docs, and if any users have these old URLS saved in bookmarks, browser history, etc.